lib/flex.js

```diff
@@ -0,0 +1,132 @@
+const Configuration = require('./configuration/configuration')
+const extractValidationStats = require('./stats/extractValidationStats')
+
+const SUPPORTED_VALIDATOR = [
+  'title',
+  'label',
+  'description',
+  'assignee',
+  'dependent',
+  'milestone',
+  'approval'
+]
+
+const SUPPORTED_ACTIONS = [
```

Comment:
In the long run I would rather we read this from a path thst can be overriden but I think these constants work for now.

Reply:
Also we don't really need these arrays. You could just catch the error of the require statement and take appropriate actions.

Reply:
yeah, try and catch would be better

Reply:
done